### PR TITLE
[ADD] add third theme : 'Système'

### PR DESCRIPTION
**Pull request by hugolopespinto**
i added a third theme + changed the way of switching theme, (toggleTheme to setTheme)

https://github.com/H-chauvet/RISU/assets/72020673/a91aa439-813c-4415-828d-bc6eef4fad47

https://github.com/H-chauvet/RISU/assets/72020673/ba8e94de-068a-4dac-987c-9a4b3e5476c8

### CODE REVIEW

File: Application/mobile/app_code/lib/main.dart [hunk 1]
```diff
@@ -6,7 +6,8 @@ import 'package:shared_preferences/shared_preferences.dart';
 import 'package:flutter_stripe/flutter_stripe.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 
-bool isDarkTheme = false;
+//bool isDarkTheme = false;
```
Comment (H-chauvet) [inline Application/mobile/app_code/lib/main.dart:9]: Remove useless comment

File: Application/mobile/app_code/lib/main.dart [hunk 2]
```diff
@@ -19,11 +20,12 @@ void main() async {
 
   // Continue with SharedPreferences and ThemeProvider
   final prefs = await SharedPreferences.getInstance();
-  isDarkTheme = prefs.getBool('isDarkTheme') ?? false;
+  //isDarkTheme = prefs.getBool('isDarkTheme') ?? false;
```
Comment (H-chauvet) [inline Application/mobile/app_code/lib/main.dart:23]: Same here, remove useless comment

**Review by H-chauvet — CHANGES_REQUESTED** [review]
Make the requested changes and you can submit again your code

File: Application/mobile/app_code/lib/main.dart [hunk 3]
```diff
@@ -6,7 +6,8 @@ import 'package:shared_preferences/shared_preferences.dart';
 import 'package:flutter_stripe/flutter_stripe.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 
-bool isDarkTheme = false;
+//bool isDarkTheme = false;
+String appTheme = 'Clair';
```
Comment (tbellicha) [inline Application/mobile/app_code/lib/main.dart:10]: Don't use a String, use an enum or a Map

File: Application/mobile/app_code/lib/pages/settings/settings_pages/theme/theme_settings_page.dart [hunk 4]
```diff
@@ -3,13 +3,30 @@ import 'package:risu/utils/theme.dart';
 import 'package:provider/provider.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
-class ThemeChangeModalContent extends StatelessWidget {
-  const ThemeChangeModalContent();
+class ThemeChangeModalContent extends StatefulWidget {
+  const ThemeChangeModalContent({super.key});
```
Comment (tbellicha) [inline Application/mobile/app_code/lib/pages/settings/settings_pages/theme/theme_settings_page.dart:7]: Put a comma after the key

**Review by tbellicha — CHANGES_REQUESTED** [review]

**Review by H-chauvet — APPROVED** [review]
Nothing wrong for me

**Pull request merged**